docs/build-the-docs.md

```diff
@@ -20,29 +20,28 @@ docker run -ti --rm \
     -e DOCSEARCH_ENGINE=lunr \
     -v $(pwd):/antora/ \
     -w /antora/ \
-    antora:custom \
-      --pull \
-      --cache-dir /antora/cache/ \
-      --redirect-facility static \
-      --generator ./generators/search.js \
-      --stacktrace \
-      site.yml
+    owncloudci/antora:latest \
+    --pull \
+    --cache-dir /antora/cache/ \
+    --redirect-facility static \
+    --generator ./generators/search.js \
+    --stacktrace \
+    site.yml
 ```
 
 This command:
 
-- Starts up [ownCloud's custom Antora Docker container](https://hub.docker.com/r/antora/antora/)
-- Removes any existing documentation
+- Starts up [ownCloud's custom Antora Docker container](https://hub.docker.com/r/owncloudci/antora/)
 - Runs Antora's `generate` command, which regenerates the documentation
-- You can add the `--clean` option to clean the build directory of any leftover artifacts from the previous build, including pdf's.
+- You can add the `--clean` option to clean the build directory of any leftover artifacts from the previous build, including ODF's.
```

Comment:
I guess you meant `ODF's` --> `PDF's`

Reply:
I'm guessing so, as nothing generates ODF files.